gematria/testing/python/basic_blocks_with_throughput.py

```diff
@@ -17,15 +17,16 @@
 import copy
 import itertools
 import os
-import unittest
+
+from google.protobuf import text_format
+from runfiles import runfiles
+from absl.testing import absltest
```

Comment:
We probably need to add this to the `BUILD` file, no?

Reply:
Not on the open source side. It's a dependency installed through `pip`, so nothing in the `BUILD.bazel` is needed here.